docs/website/docs/v1.x/dsl/request/index.md

```diff
@@ -1 +1,58 @@
-# Work in progress
\ No newline at end of file
+# Request
+
+`Request` is designed in **ZIO HTTP** in the simplest way possible to make HTTP calls.
+ It supports all HTTP request methods (as defined in [RFC2616](https://datatracker.ietf.org/doc/html/rfc2616) ) and headers along with custom methods and headers.
+ 
+ ## Creating a Request
+`Request` can be created with `method`, `url`, `headers`, `remoteAddress` and `data`.
+
+The below snippet creates a request with default params, `method` as `Method.GET`, `url` as `URL.root`, `headers` as `Headers.empty`, `data` as `HttpData.Empty`, `remoteAddress` as `None`
+```scala
+val request: Request = Request()
+```
+ ## Creating a Request Effectfully
+ You can create request effectfully using `make` constructor in `Request`.
+ 
+ The below snippet creates an effectful request with default params, method as `Method.GET`, url as `URL.root`, headers as `Headers.empty`, data as `HttpData.Empty`, remoteAddress as `None`
```

Comment:
This Part I believe will never be used on Server- Side apart from unit test and for Client side we have `clientParams` lets discuss this